Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -65,7 +65,8 @@ COREGRAPHICS_EXPORT HRESULT _DWriteCreateTextFormatWithFontNameAndSize(CFStringR
 
 // DWriteFont getters that convert to a CF/CG object or struct
 COREGRAPHICS_EXPORT CFStringRef _DWriteFontCopyInformationalString(const Microsoft::WRL::ComPtr<IDWriteFontFace>& fontFace,
```

Comment:
given these are private methods, this should make life easier.

---
In reply to: [102774998](https://github.com/Microsoft/WinObjC/pull/2058#discussion_r102774998) [](ancestors = 102774998)